android/src/main/java/com/vydia/UploaderModule.java

```diff
@@ -177,13 +177,8 @@ public void onCancelled(Context context, UploadInfo uploadInfo) {
       HttpUploadRequest<?> request;
 
       if (requestType.equals("raw")) {
-        if (customUploadId != null) {
-          request = new BinaryUploadRequest(this.getReactApplicationContext(), customUploadId, url)
-                  .setFileToUpload(filePath);
-        } else {
-          request = new BinaryUploadRequest(this.getReactApplicationContext(), url)
-                  .setFileToUpload(filePath);
-        }
+        request = new BinaryUploadRequest(this.getReactApplicationContext(), customUploadId, url)
```

Comment:
Because `String` can be `null` in Java

![image](https://user-images.githubusercontent.com/572096/30911393-58cadae8-a3c3-11e7-9437-f1ab87aedd74.png)